uqcs/templates/form.mako

```diff
@@ -1,176 +1,243 @@
+              <input type="radio" value="postgrad" name="degreeType" /> Postgraduate
+            </label>
+          </div>
+        </div>
+        <div class="form-group">
+          <label>Year</label> <br />
+          <div class="btn-group btn-group-toggle" data-toggle="buttons">
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="1" /> 1
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="2" /> 2
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="3" /> 3
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="4" /> 4
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="5" /> 5
+            </label>
+          </div>
+        </div>
       </div>
+      <input type="hidden" name="stripeToken" value="" id="stripeToken" />
+      <input
+        class="btn btn-primary"
+        name="submit"
+        type="submit"
+        id="payonline_submit"
+        value="Pay Online"
+      />
+      <input
+        type="submit"
+        name="submission"
+        value="Pay Online"
+        style="display:none;"
+        id="submitbtn"
+      />
+      <input
+        class="btn btn-success"
+        name="submission"
+        type="submit"
+        value="Pay Cash"
+      />
+    </form>
+    <div class="text-muted">
+      <p></p>
+      <p>Online payments have a 40c card surcharge.</p>
     </div>
-    <div class="form-group">
-      <label>Year</label>
-      <div class="form-control">
-        <label class="radio-inline">
-          <input type="radio" name="year" value="1"> 1
-        </label>
-        <label class="radio-inline">
-          <input type="radio" name="year" value="2"> 2
-        </label>
-        <label class="radio-inline">
-          <input type="radio" name="year" value="3"> 3
-        </label>
-        <label class="radio-inline">
-          <input type="radio" name="year" value="4"> 4
-        </label>
-        <label class="radio-inline">
-          <input type="radio" name="year" value="5+"> 5+
-        </label>
-      </div>
+    <h3>Want to register with QPAY?</h3>
+    <p>
+      As nearly all clubs and societies will be required to use QPAY for 2019,
+      UQCS will offer QPAY signup as a secondary method. QPAY is not required
+      for interacting with UQCS other than co-hosted events with clubs that
+      choose to do ticketing with QPAY.
+    </p>
+    <div class="text-muted">
+      <p></p>
+      <p>
+        UQCS does not endorse QPAY, you should do your own research to see if
+        QPAY is right for you.
+      </p>
     </div>
+    <a role="button" class="btn btn-warning" href="https://joinuqcs.getqpay.com"
+      >Signup with QPAY</a
+    >
   </div>
```

Comment:
needs some <br/> at the bottom
<img width="940" alt="screen shot 2019-01-23 at 12 09 50 am" src="https://user-images.githubusercontent.com/9084563/51591941-3e958680-1ea3-11e9-9678-758129661e4e.png">

Reply:
Will add.